docs/workflow/building/libraries/README.md

```diff
@@ -14,99 +21,96 @@ The build configurations are generally defaulted based on where you are building
 
 For more details on the build configurations see [project-guidelines](../../../coding-guidelines/project-guidelines.md#build-pivots).
 
-**Note**: Before working on individual projects or test projects you **must** run `build` from the root once before beginning that work. It is also a good idea to run `build` whenever you pull a large set of unknown changes into your branch. If you invoke the build script without any actions, the default action chain `-restore -build` is executed. This means that restore and build are not implicit when invoking other actions!
+**Note**: In order to work on individual library projects or run library tests it is necessary to have built CoreCLR (aka, the "runtime"). To do this you must run `build.sh` (or `build.cmd`) from the root once before beginning that work.
```

Comment:
If someone just came into this `building/libraries` doc without reading the overall `workflow/README`, they wouldn't be put into the "pit of success" here because they would just run `build.cmd` from the root, which would give them a debug coreclr. In my experience, it was impossible to work with a debug coreclr and try working with the libraries.